### PR TITLE
add kate's super easy spag bol recipe

### PR DESCRIPTION
**Pull request by the-hazelnut**

### CODE REVIEW

File: main_courses/0-easy/kates_super_easy_spag_bol.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+ - 1 stock cube
+ - seasoning (onion granules suggested)
+ - 400g spaghetti
+
+## Method
+
+1. Heat up a pan with olive oil. Keep the heat at medium throughout. 
+2. Add the beef mince and brown, mixing the mince until fully browned. 
+3. Add the chopped tomatoes, tomato purée, and seasoning while stirring. 
+4. Make up around 400ml of stock with the stock cube, then add to the pan.
```
Comment (neanias) [inline main_courses/0-easy/kates_super_easy_spag_bol.md]:
```suggestion
4. Dissolve the stock cube in around 400ml of hot water, then add to the pan.
```

File: main_courses/0-easy/kates_super_easy_spag_bol.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+ - 400g spaghetti
+
+## Method
+
+1. Heat up a pan with olive oil. Keep the heat at medium throughout. 
+2. Add the beef mince and brown, mixing the mince until fully browned. 
+3. Add the chopped tomatoes, tomato purée, and seasoning while stirring. 
+4. Make up around 400ml of stock with the stock cube, then add to the pan. 
+5. Stir in fully and put a lid on the pan. 
+6. Simmer for around 45 minutes or until thick. Make sure the temperature is consistent and it won't boil over for the first while; thereafter you don't need to pay particularly close attention to it.
```
Comment (neanias) [inline main_courses/0-easy/kates_super_easy_spag_bol.md:27]:
```suggestion
6. Simmer for around 45 minutes or until thick. Keep it at a medium heat, stirring occasionally. It shouldn't boil over, so you can leave it be.
```

**Review by neanias — COMMENTED** [review]
Looks good! Left a few editorial notes to make it a bit clearer/easier to understand for first time cooks

**Comment by the-hazelnut** [comment]
oops - this should probably have gone in another pull request but I'm not quite sure how this happened

File: desserts/0-easy/kates_super_easy_chocolate_cheesecake.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+### Base
+1. Line a springform tin with baking paper. 
+2. Blitz/crush the digestive biscuits, in a food processor or otherwise. 
+3. Melt the butter e.g. in a microwave. Do not overheat it. 
+4. Mix the digestive biscuits and the butter (in the springform tin or otherwise) and press flat with a spoon. 
+5. Chill the base in the refrigerator for 30 minutes. 
+
+### Cheesecake mixture
+6. Prepare a pot of hot water and set a heat-resistant bowl in it, making sure the bowl does not make contact with the water (double-boiling). Set the heat to a medium heat initially. 
+7. Decrease the heat to a low heat, and break the chocolate into small pieces and melt in the bowl when ready.
```
Comment (neanias) [inline desserts/0-easy/kates_super_easy_chocolate_cheesecake.md]:
```suggestion
7. Once boiling, decrease the heat to a low heat, and break the chocolate into small pieces and melt in the bowl. 
```

**Review by neanias — CHANGES_REQUESTED** [review]
One more edit, then happy to merge!

**Comment by the-hazelnut** [comment]
Should have made the change now, not sure why it's not picking up on the change

**Review by neanias — APPROVED** [review]

**Comment by yutotakano** [comment]
Sounds good to merge then :)

**Pull request merged**